plugin/sampling/strategystore/adaptive/processor.go

```diff
@@ -71,11 +69,11 @@ type throughputBucket struct {
 	endTime    time.Time
 }
 
-// Processor retrieves service throughput over a look back interval and calculates sampling probabilities
+// PostAggregator retrieves service throughput over a look back interval and calculates sampling probabilities
 // per operation such that each operation is sampled at a specified target QPS. It achieves this by
 // retrieving discrete buckets of operation throughput and doing a weighted average of the throughput
 // and generating a probability to match the targetQPS.
-type Processor struct {
+type PostAggregator struct {
```

Comment:
nit: I suggest we just keep the name Processor for now, to reduce the size of the diff (we can do another PR later with just the renaming).